zio-examples/jvm/src/test/scala/examples/PredicateExampleSpec.scala

```diff
@@ -0,0 +1,58 @@
+package examples
+
+
+import zio.test.{test, _}
+
+object PredicatesExampleSpecRunner extends DefaultRunnableSpec(PredicateExampleSpec.spec1)
+
+
+object PredicateExampleSpec {
+
+  val spec1: ZSpec[Any, Nothing, String] =  test("(1 + 1) must be 2") {
+    assert[Int](1 + 1, Predicate.equals(2))
+  }
+
+  val spec2: Spec[String, ZTest[Any, Nothing]] = suite("That")(spec1)
+
+  val spec: Spec[String, ZTest[Any, Nothing]] = suite("Operations")(
+      test("(1 + 1) must be 2") {
```

Comment:
Instead of the labels repeating the tests, I'd make them more high-level, e.g., this one could be called "addition operator", because that's what it is testing.